.gitignore

```diff
@@ -6,4 +6,6 @@ build/*
 !build/ploticon.js
 !build/README.md
 
+devtools/test_dashboard/mocks.json
```

Comment:
can you put that `mocks.json` in `build/` instead? 

I've been trying to be consistent by putting all built files in `build/`